michael/src/index.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import { render } from 'react-dom';
+import configureStore from './store/configureStore';
+import SomeComponent from './components/SomeComponent/SomeComponent';
+import './styles/styles.css';
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
+const store = configureStore();
+
+render(
+  <Provider store={store}>
+      <SomeComponent />
+  </Provider>,
+  document.getElementById('app')
```

Comment:
This is the problem - You're telling the document, get the component with the `app` id, but that doesn't exist on our /public/index.html. 

Change line 13 back to `document.getElementById('root');`

What this is saying is, get the element with the id root in our HTML. So if you go to /public/index.html, you'll notice that we have a `<div id="root"> </div>`which is what line 13 on this index.js is selecting for.

You also have another problem: right now you're loading `<SomeComponent/>` which doesn't exist (if you look at where you're importing SomeComponent, you're importing it from ./components/SomeComponent/SomeComponent which I don't think exists in your repository.

Reply:
For the second problem, be sure to call the `<App/>` component instead of `<SomeComponent/>`